convert2rhel/utils/__init__.py

```diff
@@ -1115,3 +1116,20 @@ def write_json_object_to_file(path, data, mode=0o600):
     with open(path, mode="w") as handler:
         os.chmod(path, mode)
         json.dump(data, handler, indent=4)
+
+
+def warn_deprecated_env(env_name):
+    """Warn the user that the environment variable is deprecated.
+
+    :param env_name: The name of the environment variable that is deprecated.
+    :type env_name: str
+    """
+    if env_name not in os.environ:
+        # Nothing to do here.
+        return
+
+    root_logger.warning(
+        "The environment variable {} is deprecated and is set to be removed on Convert2RHEL 2.4.0.\n"
+        "Please, use the configuration file instead."
+    )
+    return True
```

Comment:
If we want to use it as a getting environment variable and returning it we can do this
```suggestion
    return env_name
```

Reply:
updated to return the os.getenv(env_name.)